src/main/java/uk/gov/justice/laa/crime/cfecrime/utils/InitMeansTestOutcomeCalculator.java

```diff
@@ -3,29 +3,29 @@
 import lombok.experimental.UtilityClass;
 import lombok.extern.slf4j.Slf4j;
 import uk.gov.justice.laa.crime.cfecrime.Exceptions.UndefinedOutcomeException;
-import uk.gov.justice.laa.crime.cfecrime.cma.enums.MeansTestOutcome;
+import uk.gov.justice.laa.crime.cfecrime.enums.Outcome;
 import uk.gov.justice.laa.crime.meansassessment.staticdata.enums.InitAssessmentResult;
 
+import java.util.Objects;
+
 
 @Slf4j
 @UtilityClass
 public class InitMeansTestOutcomeCalculator {
 
-    public static MeansTestOutcome getInitMeansTestOutcome(InitAssessmentResult initAssessmentResult, boolean fullAssessmentAvailable) throws UndefinedOutcomeException {
+    public static Outcome getInitMeansTestOutcome(InitAssessmentResult initAssessmentResult, boolean fullAssessmentAvailable) throws UndefinedOutcomeException {
+
         log.debug("InitMeansTestOutcome start. Inputs: initAssessmentResult = {} fullAssessmentAvailable = {} ");
 
-        MeansTestOutcome meansTestOutcome = null;
-
-        if (initAssessmentResult == null) {
-            throw new RuntimeException("InitMeansTestOutcome: Input initAssessmentResult is null.");
-        } else {
-            if (initAssessmentResult.equals(InitAssessmentResult.FAIL) && !fullAssessmentAvailable) {
-                meansTestOutcome = MeansTestOutcome.INELIGIBLE;
-            }else if (initAssessmentResult.equals(InitAssessmentResult.PASS)) {
-                meansTestOutcome = MeansTestOutcome.ELIGIBLE_WITH_NO_CONTRIBUTION;
-            } else if (!fullAssessmentAvailable) {
-                throw new UndefinedOutcomeException( initAssessmentResult,fullAssessmentAvailable);
-            }
+        Outcome meansTestOutcome = null;
+        Objects.requireNonNull(initAssessmentResult, "InitAssessmentResult cannot be null");
+
```

Comment:
I'm confused over equals vs == here. == works (and we use it sometimes) so is there a reason to use equals()? [especially as we can see from the code that its an enum)